src/Core/Prefab/Planar/PlanarLayer.js

```diff
@@ -46,7 +46,7 @@ class PlanarLayer extends TiledGeometryLayer {
             globalExtentTMS.set(extent.crs, extent);
         }
         config.tileMatrixSets = tileMatrixSets;
-        super(id, object3d || new THREE.Group(), [extent], new PlanarTileBuilder(), config);
+        super(id, object3d || new THREE.Group(), [extent], new PlanarTileBuilder({ projection: extent.crs }), config);
```

Comment:
You can also modify the `PlanarTileBuilder` to have a mandatory parameter, `projection`, and then do `new PlanarTileBuilder(projection, {});`.